filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```diff
@@ -452,6 +452,10 @@ filebeat.inputs:
   # Available options: since_first_start, since_last_start.
   #ignore_inactive: ""
 
+  # If `take_over` is set to `true`, this `filestream` will take over all files
+  # from `log` inputs if they match at least one of the `paths` set in the `filestream`.
+  #take_over: false
```

Comment:
If this will initially be released in Beta we should also state this directly in the reference configuration.